cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraDaemonProcess.java

```diff
@@ -431,10 +431,17 @@ public void drain()
      * @throws ExecutionException   If execution is interrupted.
      * @throws IOException          If communication with Cassandra fails.
      */
-    public void upgradeTables()
+    public void upgradeSSTables(String keyspace, List<String> columnFamilies)
```

Comment:
This method, should be a part of UpgradeSSTables.java in cassandra-executor, where we pass cassandradaemonprocess as a part of constructor and get probe there. We should not add this code piece in this class